src/ophyd_epics_devices/panda.py

```diff
@@ -155,11 +159,15 @@ async def _make_block(self, name: str, num: int, block_pv: str, sim: bool = Fals
         """
         block = self.verify_block(name, num)
 
-        field_annos = get_type_hints(block)
+        field_annos = get_type_hints_no_inheritance(type(block))
         block_pvi = await pvi_get(block_pv) if not sim else None
 
         # finds which fields this class actually has, e.g. delay, width...
         for sig_name, sig_type in field_annos.items():
+            # arm PV not currently implemented
+            if sig_name == "arm":
+                continue
```

Comment:
There is another PR currently open which will fix this. As a result, this should not be committed to main.

Perhaps I'll merge my PR first, and then you can rebase on top of it and check everything still works. But we definitely shouldn't be disabling functionality (e.g. the "arm") that doesn't work when we could just delete it, which is what my PR does (and it replaces it with something that hopefully will work on a real panda)

Reply:
Should be done now